### PR TITLE
Revamp the bus view

### LINKED ISSUE #199: Add Express bus routes
The bus times that are currently shown do not account for the express bus, which runs to a limited number of stops. We should find a way to display the express bus data, either in another tab or just another list on the bus tab of the transportation section.

### PR DESCRIPTION
**Pull request by hawkrives**
Supercedes #201.
Closes #199.

Before:

After:

screen shot 2016-09-04 at 2 18 02 am
screen shot 2016-09-04 at 2 18 06 am
screen shot 2016-09-04 at 2 19 08 am

This PR changes the data format for bus times, adds express route data, and revamps the bus view in the app.

It has worked well thus far in my testing. The view updates every second, so the dots will update in real-time. All data is local to the device. 

What else to say…

Rows that the current bus line doesn't stop at are grayed-out, and their dot is hidden. Subsequent route times are shown to the right of the current time, which is the leftmost time shown (directly under the stop name.)

### CODE REVIEW

**Comment by elijahverdoorn** [comment]
Wow. This is outstanding, one of the best UIs we have by far.

**Comment by hawkrives** [comment]
Thanks! It's changing again 😆 
screen shot 2016-09-04 at 11 59 02 am

**Comment by hawkrives** [comment]
(ignore the times on the headers – they're for debugging)

File: data/bus-times.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "schedules": [
       {
         "days": ["mo", "tu", "we", "th", "fr", "sa"],
-        "stops": ["St. Olaf", "Carleton", "Food Co-op", "Cub/Target", "El Taquila", "Food Co-op", "Carleton", "St. Olaf"],
+        "stops": ["St. Olaf", "Carleton", "Food Co-op", "Cub/Target", "El Tequila", "Food Co-op", "Carleton", "St. Olaf"],
```
Comment (hawkrives) [inline data/bus-times.json:7]: I will have you know that I copied that straight out of their official map brochure thing. ;P

Comment (elijahverdoorn) [reply data/bus-times.json:7]: That's funny, made better since I was there literally an hour ago.

**Comment by hawkrives** [comment]
screen shot 2016-09-04 at 12 03 31 pm

**Comment by hawkrives** [comment]
Alright, I think it's ready to merge.

The current stop (any stop that is at the current minute) is bolded; stops that have been passed are grayed out, as are stops that will be skipped.

The section titles will say "Running", "Last Bus", "Over for Today", and "Starting 8:05am", at various times.

screen shot 2016-09-04 at 1 44 48 pm
screen shot 2016-09-04 at 1 44 42 pm
screen shot 2016-09-04 at 1 43 39 pm

**Comment by drewvolz** [comment]
Wow @hawkrives, incredible UI work!

**Pull request merged**